ragtime.core/src/ragtime/main.clj

```diff
@@ -27,16 +27,34 @@
   (map verbose-migration ((load-var migration-fn))))
 
 (defn migrate [{:keys [database migrations]}]
-  (core/migrate-all
-   (core/connection database)
-   (resolve-migrations migrations)))
+  (letfn [(migrate_db [the_db]
+                      (core/migrate-all
+                        (core/connection the_db)
+                        (resolve-migrations migrations))
+                      )
+          ]
+    (if (coll? database)
+      (doseq [db database]
+        (migrate_db db)
+        )
+      (migrate_db database)
+      )))
```

Comment:
I am making the pull request first time.
I  have changed the code style .Need making a pull request again?